### ISSUE TITLE
"Unable to validate the following destination configurations" on "S3Bucket" resource in CFT

### ISSUE THREAD

**Issue by andrejmaya** [aws-samples/aws-control-tower-account-factory#1]
When I deploy the `control-tower-account-factory-solution.yml` CFT with `CreateConfigurationBucket` parameter set to true I get an `Unable to validate the following destination configurations` error message and the CFT deployment fails.

The reason is that the `AccountFactoryFunction` Lambda Function must be created before the Bucket gets created.

**Comment by RemekHet** [comment]
That should be necessary. I added lambda policy to deployment CF template. That should solve your problem. Please check it and let me know

**Comment by andrejmaya** [comment]
Hi @RemekHet, unfortunately the Permission only won't help. Have you saw my pull request https://github.com/aws-samples/aws-control-tower-account-factory/pull/2 ? This solves the problem

**Comment by RemekHet** [comment]
I tested couple time and it works for me. I'm not sure how your solution solves problem - if I get this correctly you just removed notification from bucket creation. If so, how you adding notification to bucket?

**Comment by andrejmaya** [comment]
yes that's fine. Check out my other pull request here: https://github.com/aws-samples/aws-control-tower-account-factory/pull/4

Otherwise you can start CFT with empty parameters
